packages/corvid-cli/src/apps/clone.js

```diff
@@ -4,7 +4,7 @@ const chalk = require("chalk");
 const normalize = require("normalize-url");
 const { URL } = require("url");
 const getMessage = require("../messages");
-const { UserError } = require("corvid-local-logger");
+const logger = require("corvid-local-logger");
```

Comment:
FYI, you can do:
const {logger, UserError} = require('.....')

Reply:
done